addons/calendar_sms/migrations/13.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,14 @@
+---Models in module 'calendar_sms'---
+
+---Fields in module 'calendar_sms'---
+calendar_sms / calendar.alarm           / alarm_type (False)            : NEW selection_keys: ['email', 'notification', 'sms'], mode: modify
+calendar_sms / calendar.alarm           / type (False)                  : DEL selection_keys: ['email', 'notification', 'sms'], mode: modify
+# DONE: pre-migration: try to rename fields: calendar module already renames this field
```

Comment:
I think enough to write `NOTHING TO DO: Done in Calendar module` and remove the pre-migration script.

Reply:
Done.